ledger/src/blockstore.rs

```diff
@@ -10787,38 +10787,70 @@ pub mod tests {
         assert!(!blockstore.is_dead(0));
     }
 
+    /// Prepare two FEC sets of shreds for the same slot index
+    /// with reasonable shred indices, but in such a way that
+    /// both FEC sets include a shred with LAST_IN_SLOT flag set.
+    fn setup_duplicate_last_in_slot(
+        slot: Slot,
+    ) -> ((Vec<Shred>, Vec<Shred>), (Vec<Shred>, Vec<Shred>)) {
+        let entries = make_slot_entries_with_transactions(1);
+        let leader_keypair = Arc::new(Keypair::new());
+        let reed_solomon_cache = ReedSolomonCache::default();
+        let shredder = Shredder::new(slot, 0, 0, 0).unwrap();
+        let (shreds1, code1) = shredder.entries_to_shreds(
```

Comment:
nit: Shall we use `Shredder::make_merkle_shreds_from_entries()` instead since we intend to kill `entries_to_shred()` (or at least its' current form with `merkle_variant` argument) ?